CONTRIBUTING.md

```diff
@@ -190,19 +190,62 @@ make
 
 ### Compiler Options
 
-By default, when building project with no options, next static libraries are generated
+By default, when building project with no options, next static libraries are generated:
 
 - ``Libraries``:
   - az_core
+    - az_span, az_http, az_json, etc.
   - az_iot
-  - az_keyvault
+    -  iot_provisioning, iot_hub, etc.
   - az_storage_blobs
-- ``Platform Abstraction Layer``: Default empty implementation for platform functions like time and http stack. This default implementation is used to compile only but will return ERROR NOT IMPLEMENTED when running it.
+    -  Storage SDK blobs client.
   - az_noplatform
+    - Library that provides a basic returning error for platform abstraction as AZ_NOT_IMPLEMENTED. This ensures project can be compiled without the need to provide any specific platform implementation. This is useful if you want to use az_core without platform specific functions likes Mutex or Time. 
   - az_nohttp
-  - az_posix (on Lin/Mac)
```

Comment:
Why are these dropped but az_notplatform stays?

Reply:
because az_notttp is generated/compiled with default compiler-options.  
az_posix and az_win32 are mentioned below with the required compiler-option